src/components/Footer/index.tsx

```diff
@@ -1,146 +1,91 @@
 import styles from './styles.module.css';
 import {
+  faTwitter,
   faGithubAlt,
   faLinkedin,
   faTwitch,
-  faTwitter,
   faYoutube,
+  faInstagram,
 } from '@fortawesome/free-brands-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import Link from 'next/link';
-import { faHeart } from '@fortawesome/free-solid-svg-icons';
 
 const Footer: React.FC = () => {
-  return (
-    <footer className="relative bg-gray-800">
-      <img
-        className={`absolute w-20 ${styles.impostor}`}
-        src="/img/impostor.png"
-      />
-      <div
-        className="bottom-auto top-0 left-0 right-0 w-full absolute pointer-events-none overflow-hidden -mt-20"
-        style={{ height: '80px', transform: 'translateZ(0)' }}
-      >
-        <svg
-          className="absolute bottom-0 overflow-hidden"
-          xmlns="http://www.w3.org/2000/svg"
-          preserveAspectRatio="none"
-          version="1.1"
-          viewBox="0 0 2560 100"
-          x="0"
-          y="0"
-        >
-          <polygon
-            className="text-gray-800 fill-current"
-            points="2560 0 2560 100 0 100"
-          ></polygon>
-        </svg>
-      </div>
-      <div className="container mx-auto px-4 p-8 pb-6">
-        <div className="flex flex-wrap">
-          <div className="w-full lg:w-6/12 px-4 text-center md:text-left">
-            <h4 className="text-2xl font-semibold text-white">
-              Buscanos en nuestras redes
-            </h4>
-            <div className="flex space-x-1 mt-6 justify-center md:justify-start">
-              <Link href="https://twitter.com/FrontEndCafe">
-                <a
-                  target="_blank"
-                  className="grid place-items-center h-10 w-10 rounded-full bg-white text-blue-500 hover:bg-gray-200"
-                >
-                  <FontAwesomeIcon icon={faTwitter} size="lg" />
-                </a>
-              </Link>
-
-              <Link href="http://github.com/frontend-cafe">
-                <a
-                  target="_blank"
-                  className="grid place-items-center h-10 w-10 rounded-full bg-white text-gray-900 hover:bg-gray-200"
-                >
-                  <FontAwesomeIcon icon={faGithubAlt} size="lg" />
-                </a>
-              </Link>
+   const navItems = [
```

Comment:
Can we move this array outside of the Footer component since it doesn't need anything from it?